docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
+=============
+
+.NET is a very flexible and naturally cross-platform [specification](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) for building developer products. It is used for all of the most popular app categories: desktop, mobile, cloud, gaming and IoT. 
+
+Major commercial and open source .NET products are listed below, in order of introduction.
+
+.NET Framework
+--------------
+
+The .NET Framework is Microsoft's primary develop platform for Windows server and client developers. It is a very powerful and mature framework, with a large class library that supports a wide variety of applications and solutions on Windows.
+
+There are multiple application models built on top of the .NET Framework that allow developers to build applications ranging from console applications, across rich client (WPF) applications to scalable web applications.
+
+[Windows Forms](https://msdn.microsoft.com/library/dd30h2yb.aspx) and [Windows Presentation Foundation (WPF)](https://msdn.microsoft.com/library/ms754130.aspx) are User Interface (UI) stacks that enable you to build desktop applications for Windows. The strength of Windows Forms lies in its rich support for common databinding scenarios as well as access to Windows’ native user interface controls. WPF, on the other hand, allows you to exercise much more control over the look and feel of your application. 
+
+[Windows Communication Foundation (WCF)](https://msdn.microsoft.com/library/ms731082.aspx) is a set of libraries for SOAP Web Services. It allows you to create services that can communicate through various supported protocols using various data formats, and that can be hosted in any process you choose. This leads to one of the major features of WCF: your services are not tied to any particular hosting strategy or approach.
```

Comment:
I agree with @tdykstra WCF feels out of place, even though it supports JSON and REST now.